lib/solidus_importer/base_importer.rb

```diff
@@ -2,8 +2,17 @@
 
 module SolidusImporter
   class BaseImporter
+    attr_reader :options, :logger
+
     def initialize(options)
       @options = options
+      @logger ||= begin
+        if @options[:logger] && !@options[:logger].is_a?(Class)
+          @options[:logger]
+        else
+          (@options[:logger] || ::SolidusImporter::BaseLogger).new
```

Comment:
Any reason for not passing `Logger.new(STDOUT)` directly?

Reply:
Good point 👍 

As discussed, I'm going to move the logger replacement in another PR.

Reply:
Moved [here](https://github.com/nebulab/solidus_importer/pull/19)